v0/destinations/campaign_manager/transform.js

```diff
@@ -42,7 +42,9 @@ function buildResponse(
     requestType === "batchinsert"
       ? "dfareporting#conversionsBatchInsertRequest"
       : "dfareporting#conversionsBatchUpdateRequest";
-  response.body.JSON.encryptionInfo = encryptionInfo || {};
+  if (isDefinedAndNotNull(encryptionInfo)) {
```

Comment:
This is only required when encryptedUserId or encryptedUserIdCandidates[] is used. Need to add validation for that.
Also this should be build with mapping JSON
with the following info https://developers.google.com/doubleclick-advertisers/rest/v4/EncryptionInfo
also  `EncryptionEntityType` and `EncryptionSource` should be validated against proper enums

Reply:
check was already there , updated validation from valid posssible values.